lib/bin/lwm2m_carrier/doc/requirements.rst

```diff
@@ -25,13 +25,19 @@ Below are some of the requirements and limitations of the application while runn
 * The LwM2M carrier library controls the LTE link.
 
    * This is needed for storing keys to the modem, which requires disconnecting from an LTE link and connecting to it.
-   * The application should not connect to the LTE link between a :c:macro:`LWM2M_CARRIER_EVENT_DISCONNECTING` event and :c:macro:`LWM2M_CARRIER_EVENT_CONNECTED` event.
+   * The application cannot use the LTE link between a :c:macro:`LWM2M_CARRIER_EVENT_DISCONNECTING` event and :c:macro:`LWM2M_CARRIER_EVENT_CONNECTED` event.
+     Instead, the application must wait for the :c:macro:`LWM2M_CARRIER_EVENT_LTE_READY` event.
+
+* The LwM2M carrier library uses the modem DFU socket for FOTA.
```

Comment:
.. and a TLS socket.

I think we should mention these two together instead of separately.
Let's try to rephrase a bit here so that we mention both together: FOTA needs the DFU socket and a TLS socket. When the event is received, the application must close any TLS sockets and the DFU socket if in use.